### PR TITLE
Implement a new footer

### PR DESCRIPTION
**Pull request by fabioberger**
This PR:
- Re-designs the footer
- Replace hard-coded urls with constants

screen shot 2017-10-06 at 20 22 32

### CODE REVIEW

**Comment by BMillman19** [comment]
The footer doesn't appropriately layout for smaller screen widths

screen shot 2017-10-06 at 10 37 58 am

**Comment by fabioberger** [comment]
Ah yup, still need to mobile optimize. Good catch!

File: ts/components/footer.tsx [hunk 1]
```diff
@@ -1,83 +1,254 @@
...
-    },
-    slackIcon: {
-        backgroundColor: 'white',
-        width: 17,
-        height: 17,
-        marginTop: 1,
-        marginLeft: 9,
-    },
+const ICON_DIMENSION = 16;
+const CUSTOM_DARK_GRAY = '#393939';
```
Comment (BMillman19) [inline ts/components/footer.tsx]: maybe we should start thinking about extracting all of these into one place for color constants so a more consistent color palette is enforceable and we can more easily experiment with sweeping color changes

File: ts/components/footer.tsx [hunk 2]
```diff
@@ -1,83 +1,254 @@
...
-                            <div className="col col-3 pt1">
-                                <a href="https://github.com/0xProject" target="_blank">
-                                    <i className="zmdi zmdi-github-box" style={{...styles.icon}} />
-                                </a>
+            <div className="relative pb4 pt2" style={{backgroundColor: CUSTOM_DARK_GRAY}}>
+                <div className="mx-auto max-width-4 py4 clearfix" style={{color: 'white'}}>
+                    <div className="col col-4 left">
+                        <div className="center" style={{width: 148}}>
+                            <div>
+                                <img src="/images/protocol_logo_white.png" height="30" />
```
Comment (BMillman19) [inline ts/components/footer.tsx]: can we make the img path a constant?

File: ts/components/footer.tsx (hunk 1 above)
Comment (fabioberger) [reply ts/components/footer.tsx]: This is a great idea, out of scope for this PR though. Adding it to the TODO's in Airtable.

**Comment by fabioberger** [comment]
footer_mobile

**Pull request merged**